src/Compilers/CSharp/Test/Emit3/FlowAnalysis/RegionAnalysisTests.cs

```diff
@@ -14109,6 +14109,9 @@ ref struct RS
                 """,
                 targetFramework: TargetFramework.NetCoreApp);
             comp.VerifyEmitDiagnostics(
+                // (3,13): warning CS0649: Field 'RS.ri' is never assigned to, and will always have its default value (null reference)
```

Comment:
The wording of the diagnostic feels a little confusing. Perhaps it would make sense to introduce a new message for ref fields which indicates the need to ref-assign.

Reply:
Sounds good. Does this wording seem fine?

> Field '{0}' is never ref-assigned to, and will always have its default value (null reference)

Reply:
Btw, technically that will be a new warning that can be reported for existing code - even where it wasn't previously reported due to the bug being fixed - but also where it was reported but under a different error code. So should it be part of a new warning wave?

Reply:
> So should it be part of c# 14 warning wave?

I think the simplest, lowest risk way will be to use new wording and put in a warning wave.